src/target/renesas_rz.c

```diff
@@ -73,10 +73,16 @@
  * except for pnrs stored in 'FIXED_PNR1', where the code is stored in reverse order (but the last 3 bytes are still 0x20 aka ' ')
  */
 
-/* Base address and size for the 4 OCRAM regions + their mirrors (includes RETRAM) */
+/* Base address for the RZ-series, including mirror */
```

Comment:
Grammatical nitpicks: `/* Base address for the OCRAM regions, including their mirrors */` - the sentence needs an object for what the subject is about, and as the object us plural the follow on must be too.

If memory serves correctly, this still includes the RETRAM region so probably worth noting that.